apps/console-lite/src/app/components/deal-ticket/deal-ticket-size.tsx

```diff
@@ -6,11 +6,11 @@ import {
   SliderTrack,
   SliderRange,
   Button,
-  Input,
   FormGroup,
 } from '@vegaprotocol/ui-toolkit';
 import { BigNumber } from 'bignumber.js';
 import { DealTicketEstimates } from './deal-ticket-estimates';
+import { InputSetter } from '../input-setter/';
```

Comment:
```suggestion
import { InputSetter } from '../input-setter;
```

chore (non-blocking): this trailing slash gives me ocd, i can't stop myself from wanting to remove it :)